package.json

```diff
@@ -48,25 +49,25 @@
     "url": "git+https://github.com/bwinton/SnoozeTabs.git"
   },
   "scripts": {
-    "start": "npm run build && npm-run-all --parallel watch run storybook",
-    "start-win": "npm run build && npm run watch",
-    "build": "npm run clean && npm-run-all --parallel copy:assets bundle:js bundle:css",
-    "run": "web-ext run -s dist --firefox=firefoxdeveloperedition",
-    "clean": "rm -rf dist && mkdir dist",
-    "watch": "npm-run-all --parallel watch:js watch:css watch:assets",
-    "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && shx mkdir -p dist/popup && shx cp -r src/popup/*.html dist/popup/",
-    "bundle:js": "npm run lint:js && rollup -c --environment entry:background && rollup -c --environment entry:popup/snooze-content",
+    "build": "npm run clean && npm-run-all --parallel copy:assets bundle:*",
```

Comment:
Alphabetizing the tasks here kind of obscures the PR - not sure what the actual change was for SVG optimization. Also, the order was vaguely intentional as I was trying to keep the more important commands first in the list and roughly sort the sub-commands later in the list.

More a nit than anything, but I'd like to separate things like this into different commits